tailwind.config.js

```diff
@@ -63,6 +64,18 @@ utilityFiles.forEach((file) => {
 });
 plugins = plugins.concat(utilities);
 
+// Autocomplete focus styles for the options.
+// eslint-disable-next-line no-shadow
+const autocomplete = plugin(({ addComponents, theme }) =>
+  addComponents({
+    '.autocomplete-option[data-focus="true"]': {
+      backgroundColor: theme('colors.digital-blue.DEFAULT'),
+      color: theme('colors.white'),
+    },
+  })
+);
+plugins = plugins.concat(autocomplete);
```

Comment:
This could be a problem as we are not requiring individuals that consume this package to include the TailWindCSS confirmation from this package. Can you not get at the options to style?

Reply:
Please remove this custom TWComponent and instead use the `componentsProps` option on the MUI components to inject your class names into the nested components.

Reply:
https://mui.com/customization/unstyled-components/

Reply:
I removed it from the tailwind config, hovewer componentProps didn't work either for my case, used a different solution inside the component